api/controllers/todoListControllers.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
why is this file named 'todoListControllers.js' if it has functions that deal with clients?